src/components/puck/registry/components.ts

```diff
@@ -0,0 +1,17 @@
+import { Registry } from "./schema.ts";
+
+export const components: Registry = [
+  {
+    name: "heading",
+    type: "registry:component",
+    files: ["atoms/heading.tsx"],
+  },
+  {
+    name: "HeadingText",
+    type: "registry:component",
+    registryDependencies: [
+      "https://reliably-numerous-kit.pgsdemo.com/components/heading.json",
```

Comment:
Make the url a const so we can update it in one spot when we change it later.

Reply:
Ack, with the new changes it can just be `heading`